webapp/fmprod/src/main/java/org/apache/oodt/cas/product/jaxrs/services/FileManagerJaxrsServiceV2.java

```diff
@@ -71,9 +72,9 @@
  */
 public class FileManagerJaxrsServiceV2 {
 
-  private static Logger logger = LoggerFactory.getLogger(FileManagerJaxrsServiceV2.class);
+  private static final Logger LOGGER = Logger.getLogger(FileManagerJaxrsServiceV2.class.getName());
```

Comment:
We are trying to move to slf4j completely since java util logging is not a rich implementation. In the existing version, you have used slf4j. Could you comply to that in the updated class as well?